### PR TITLE
Fix the comment not shown after refresh issue

### PR DESCRIPTION
**Pull request by imhassantariq**
Ticket link: https://edlyio.atlassian.net/browse/EDS-153

This PR fixes the issue we are facing when the user comment on any post and after the refresh, the comment doesn't show.

### CODE REVIEW

File: common/static/common/js/discussion/utils.js [hunk 1]
```diff
@@ -206,7 +206,6 @@
                     );
                 };
             }
-
```
Comment (danialmalik) [inline common/static/common/js/discussion/utils.js]: Revert this change please.

File: common/static/common/js/discussion/utils.js [hunk 2]
```diff
@@ -220,7 +219,13 @@
 
             request = $.ajax(params).always(function() {
                 if ($elem) {
-                    $elem.prop('disabled', false);
+                    var elemClassList = $elem.content.classList;
```
Comment (danialmalik) [inline common/static/common/js/discussion/utils.js:223]: Please add a comment here telling why this change is required and this change is specific for lums-soe.

File: common/static/common/js/discussion/views/thread_response_view.js [hunk 3]
```diff
@@ -184,13 +184,13 @@
...
                 var id = event.target.id.split('-').slice(-1)[0];
-                var query_selector = 'form[data-id="'+ id +'"] .discussion-submit-comment';
-                var comment_button = document.querySelector(query_selector);
+                var querySelector = 'form[data-id="'+ id +'"] .discussion-submit-comment';
+                var commentButton = document.querySelector(querySelector);
                 if (event.target.value.length > 0) {
-                    comment_button.disabled = false;
+                    commentButton.disabled = false;
                 }
                 else {
```
Comment (danialmalik) [inline common/static/common/js/discussion/views/thread_response_view.js]:
```suggestion
               } else {
```

File: common/static/common/js/discussion/views/discussion_thread_view.js [hunk 4]
```diff
@@ -347,14 +347,14 @@
                 return this.model.set('endorsed', this.$el.find('.action-answer.is-checked').length > 0);
             };
             DiscussionThreadView.prototype.toggleResponseSubmitButton = function(event) {
-                var post_button = $('.discussion-submit-post');
+                var postButton = $('.discussion-submit-post');
                 if (event.target.value.length) {
-                    post_button.attr('disabled', false);
+                    postButton.attr('disabled', false);
                 }
                 else {
```
Comment (danialmalik) [inline common/static/common/js/discussion/views/discussion_thread_view.js]:
```suggestion
               } else {
```

**Review by danialmalik — CHANGES_REQUESTED** [review]

**Review by danialmalik — APPROVED** [review]

**Review by tehreem-sadat — APPROVED** [review]
